packages/webapp/.env.example

```diff
@@ -1,3 +1,11 @@
+# RPC API KEY
 NEXT_PUBLIC_ALCHEMY_API_KEY=
```

Comment:
These API keys shouldn't be public. `ALCHEMY_API_KEY`, `ETHERSCAN_API_KEY`, `BASESCAN_API_KEY` could all get abused by someone wanting to grief us by rate limiting the site. If there are no objections, I'll open an issue for it.

Reply:
Agreed, open an issue for it.

Reply:
Agreed

Reply:
we should be able to SSR these values into the requests